### PR TITLE
[RRFS_baseline] Add thresholding to REFC and RETOP

### PR DESCRIPTION
**Pull request by JeffBeck-NOAA**
## DESCRIPTION OF CHANGES: 
Add 15 dBZ threshold to REFC and RETOP.

### CODE REVIEW

File: ush/templates/parm/metplus/RETOP.conf [hunk 1]
```diff
@@ -74,14 +74,15 @@ GRID_STAT_REGRID_TO_GRID = FCST
...
+#FCST_VAR1_OPTIONS = convert(x) = M_to_KFT(x);
+FCST_VAR1_OPTIONS = cnt_thresh = [ >15 ]; convert(x) = M_to_KFT(x);
 
 BOTH_VAR1_THRESH = ge20, ge30, ge40, ge50
 
 OBS_VAR1_NAME = EchoTop18
 OBS_VAR1_LEVELS = Z500
-OBS_VAR1_OPTIONS = censor_thresh = eq-999; censor_val = -9999; convert(x) = KM_to_KFT(x);
-
+#OBS_VAR1_OPTIONS = censor_thresh = eq-999; censor_val = -9999; convert(x) = KM_to_KFT(x);
```
Comment (jwolff-ncar) [inline ush/templates/parm/metplus/RETOP.conf]: Go ahead and remove the old version. If we want to revert we can just take out the cnt_thresh settings.

File: ush/templates/parm/metplus/REFC.conf [hunk 2]
```diff
@@ -75,12 +75,14 @@ GRID_STAT_REGRID_TO_GRID = FCST
...
 FCST_VAR1_LEVELS = L0
+FCST_VAR1_OPTIONS = cnt_thresh = [ >15 ];
 
 BOTH_VAR1_THRESH = ge20, ge30, ge40, ge50
 
 OBS_VAR1_NAME = MergedReflectivityQComposite
 OBS_VAR1_LEVELS = L0 
-OBS_VAR1_OPTIONS = censor_thresh = eq-999; censor_val = -9999;
+OBS_VAR1_OPTIONS = censor_thresh = eq-999; censor_val = -9999; cnt_thresh = [ >15 ]; cnt_logic = UNION;
+#OBS_VAR1_OPTIONS = cnt_thresh = [ >15 ]; cnt_logic = UNION;
```
Comment (jwolff-ncar) [inline ush/templates/parm/metplus/REFC.conf]: Same comment as below (I guess I work from bottom to top!)

File: ush/templates/parm/metplus/RETOP.conf [hunk 3]
```diff
@@ -74,14 +74,15 @@ GRID_STAT_REGRID_TO_GRID = FCST
 # Forecast/Observation variable Information
 FCST_VAR1_NAME = RETOP
 FCST_VAR1_LEVELS = L0
-FCST_VAR1_OPTIONS = convert(x) = M_to_KFT(x);
+#FCST_VAR1_OPTIONS = convert(x) = M_to_KFT(x);
```
Comment (jwolff-ncar) [inline ush/templates/parm/metplus/RETOP.conf]: Same comment as below (I guess I work from bottom to top!)

**Review by jwolff-ncar — COMMENTED** [review]
As you suggested, let's go ahead and remove the comments. Otherwise, I think it looks ok.

File: ush/templates/parm/metplus/RETOP.conf (hunk 1 above)
Comment (jwolff-ncar) [reply ush/templates/parm/metplus/RETOP.conf:84]: This needs to have the cnt_thresh included though. I don't see it here. I might be missing something.

Comment (JeffBeck-NOAA) [reply ush/templates/parm/metplus/RETOP.conf]: Done!

File: ush/templates/parm/metplus/REFC.conf (hunk 2 above)
Comment (JeffBeck-NOAA) [reply ush/templates/parm/metplus/REFC.conf]: Done!

File: ush/templates/parm/metplus/RETOP.conf (hunk 3 above)
Comment (JeffBeck-NOAA) [reply ush/templates/parm/metplus/RETOP.conf]: Done!

**Review by jwolff-ncar — APPROVED** [review]

**Pull request merged**